### PR TITLE
pkp/pkp-lib#6478 Don't use CDN for fonts/script assets

### PR DESCRIPTION
**Pull request by NateWr**
@Vitaliy-1 can you double-check that this looks ok?

### CODE REVIEW

**Comment by NateWr** [comment]
Oh, I see that I will need to build the LESS files using gulp. @Vitaliy-1 would you be able to do that with this and the other themes if I prepare the changes?

**Comment by Vitaliy-1** [comment]
@NateWr, it's not necessary here. I compile with Gulp only npm dependencies. Thanks! 
I have a feeling that I've removed a dependency from `ionicons`, otherwise, I'm wondering why I'm not using Gulp for compiling it, will check.

File: ClassicThemePlugin.inc.php [hunk 1]
```diff
@@ -60,37 +60,24 @@ public function init()
 
 		// Styles for HTML galleys
 		$this->addStyle('htmlGalley', 'templates/plugins/generic/htmlArticleGalley/css/default.css', array('contexts' => 'htmlGalley'));
-		$this->addStyle(
-			'htmlFont',
-			'https://fonts.googleapis.com/css?family=Cardo:400,400i,700|Montserrat:400,400i,700,700i,900,900i',
-			array('baseUrl' => '', 'contexts' => 'htmlGalley')
-		);
+		$this->addStyle('htmlFont', 'less/fonts.less'. ['contexts' => 'htmlGalley']);
```
Comment (Vitaliy-1) [inline ClassicThemePlugin.inc.php:63]: Just a small typo here

**Comment by Vitaliy-1** [comment]
Looks good! Do you have time to correct a small typo and I'll take care of the rest?

**Comment by NateWr** [comment]
Good catch! All fixed.

**Comment by Vitaliy-1** [comment]
Thanks, merged!

**Pull request merged**